samples/wifi/sta/README.rst

```diff
@@ -23,6 +23,14 @@ This sample can perform Wi-Fi operations such as connect and disconnect in the 2
 
 Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode.
 
+Quad Serial Peripheral Interface (QSPI) encryption
+**************************************************
+
+This sample enables QSPI encryption by default using the factory default key (all F's), but you can change the key by setting the :kconfig:option:`CONFIG_NRF700X_QSPI_ENCRYPTION_KEY` Kconfig option,
+this demonstrates only the API usage.
+
+If encryption of the QSPI traffic is required for production devices, then matching keys should be programmed in both the nRF7002 OTP and non-volatile storage associated with the host.
```

Comment:
```suggestion
If encryption of the QSPI traffic is required for production devices, then matching keys should be programmed in both the nRF7002 OTP and nonvolatile storage that are associated with the host.
```

Reply:
Please ensure that this suggested edit does not change the intended meaning of this sentence.

Are both the nRF7002 OTP and nonvolatile storage associated with the host, or only the nonvolatile storage? If it is only the nonvolatile storage that is associated with the host, then leave this part of the sentence as is.

Reply:
nrf7002 is a companion chip not part of host.